style/client.css

```diff
@@ -3381,6 +3381,36 @@ a.ilink.yours {
 
 /* teambuilder set */
 
+.dark .utilichart h3, .dark .dexentry h3, .dark .resultheader h3 {
+	background: #636363;
+	color: #F1F1F1;
+	border: 1px solid #A9A9A9;
+	text-shadow: 1px 1px 0 rgb(40, 43, 45);
+	box-shadow: inset 0px 1px 0 rgb(49, 49, 49);
+	border-right: none;
+}
+
+.dark .teambar button {
+	background: #5A6570;
+	color: #F1F1F1;
+	border: 1px solid #AAAAAA;
+}
+
+.dark .teambar button:hover {
+	background: #444C54;
+	color: #F1F1F1;
+	border: 1px solid #AAAAAA;
+}
+
+.dark .teambar button:disabled, .dark .teambar button:disabled:hover, .dark .teambar button:disabled:active {
+    color: #ffffff;
+    background: #2d343a;
+    border-color: #6bacc5;
+    opacity: 1;
+}
+
+
```

Comment:
I'd suggest removing 2 of these blank lines (1 blank line is good for organizational purposes, other than that it becomes a bit of an annoyance to read)